.github/workflows/android_deploy_firebase.yml

```diff
@@ -0,0 +1,50 @@
+name: Android - Deploy production build to Firebase
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_and_deploy_android:
+    name: Build & Deploy Android
+    runs-on: ubuntu-latest
+    environment: staging
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+
+      - name: Setup Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+
+      - name: Get flutter dependencies
+        run: flutter pub get
+
+      - name: Run code generator.
+        run: flutter packages pub run build_runner build --delete-conflicting-outputs
+
+      - name: Setup .env
+        env:
+          ENV_PRODUCTION: ${{ secrets.ENV_PRODUCTION }}
+        run: |
+          echo $ENV_PRODUCTION > .env
+
+      - name: Build Android apk
+        run: flutter build apk --flavor production --debug --build-number $GITHUB_RUN_NUMBER
+
+      - name: Deploy Android to Firebase
+        uses: wzieba/Firebase-Distribution-Github-Action@v1.5.0
+        with:
+          appId: ${{ secrets.FIREBASE_ANDROID_APP_ID_PRODUCTION }}
+          token: ${{secrets.FIREBASE_TOKEN}}
```

Comment:
Let me update this to the new approach: `serviceCredentialsFileContent`
Ref: https://github.com/wzieba/Firebase-Distribution-Github-Action#firebase-app-distribution-github-action

Reply:
@ryan-conway It seems iOS template still use this `FIREBASE_TOKEN`. So I think we can leave it as-is to make all CD scripts work first. 🙈